rpcs3/Input/pad_thread.cpp

```diff
@@ -290,8 +296,10 @@ void pad_thread::operator()()
 
 	while (thread_ctrl::state() != thread_state::aborting)
 	{
-		if (!pad::g_enabled || Emu.IsPaused() || !is_input_allowed())
+		if (!pad::g_enabled || !is_input_allowed())
```

Comment:
This will add a constant nontrivial CPU load which is to be discouraged

Reply:
I've actually increased sleep time here so it would be reduced, but then again this is 30ms sleep loops so the CPU load here is ignorable.